gazebo_plugins/src/gazebo_ros_diff_drive.cpp

```diff
@@ -475,13 +477,14 @@ void GazeboRosDiffDrive::UpdateOdometryEncoder()
         //  updating. Results in a 0 speed signal.
```

Comment:
Please update this comment to reflect your changes. The lack of encoder updates should still be presenting a 0 update value to the joint states. Since the joint states are doing their own estimation, that's probably where the noise originates. I would just like the comment to be consistent with the code.